src/number.rs

```diff
@@ -8,17 +14,34 @@ pub const SYS_ARG_MSLICE: usize=0x0200_0000;
 pub const SYS_ARG_PATH: usize = 0x0300_0000;
 
 pub const SYS_RET: usize =      0x00F0_0000;
+/// Flag: This syscall returns a file descriptor.
 pub const SYS_RET_FILE: usize = 0x0010_0000;
 
+/// --- Syscalls
+
 pub const SYS_LINK: usize =     SYS_CLASS_PATH | SYS_ARG_PATH | 9;
 pub const SYS_OPEN: usize =     SYS_CLASS_PATH | SYS_RET_FILE | 5;
 pub const SYS_CHMOD: usize =    SYS_CLASS_PATH | 15;
 pub const SYS_RMDIR: usize =    SYS_CLASS_PATH | 84;
 pub const SYS_UNLINK: usize =   SYS_CLASS_PATH | 10;
+/// Syscall: Bind a file descriptor exposed by a process to a local file descriptor.
+///
+/// The file descriptor must already have been exposed through a call to `SYS_DUP_EXPORT`..
+///
+/// Prototype: `sys_dup_from(path, pid) -> fd`
+pub const SYS_DUP_FROM: usize = SYS_CLASS_PATH | 11;
```

Comment:
I prefer numbers higher than 500 for Redox specific syscalls (even though the class is different). 11 is execve.